phpstan.neon

```diff
@@ -0,0 +1,16 @@
+parameters:
+    level: 5
+    paths:
+        - Config/
+        - Exceptions/
+        - Formatter/
+        - Language/
+        - Matcher/
+        - Parser/
+        - Styles/
+        - Utils/
+
+    ignoreErrors:
+        -
+            message: '#Variable \$match in empty\(\) always exists and is not falsy.#'
+            path: Matcher/RegexMatcher.php
```

Comment:
@kadet1090 Normally I trust phpstan with my life xD but here I didn't want to make a change in the code without having your confirmation. I'm not sure how the heck would it know that the group's match is never empty/falsy